test/ec_internal_test.c

```diff
@@ -433,6 +434,90 @@ int ecpkparams_i2d2i_test(int n)
     return testresult;
 }
 
+
+static int check_bn_mont_ctx(BN_MONT_CTX *mont, BIGNUM *mod, BN_CTX *ctx)
+{
+    int ret = 0;
+    BN_MONT_CTX *regenerated = BN_MONT_CTX_new();
+
+    if (!TEST_ptr(regenerated)) {
+       return ret;
+    }
+    if (!TEST_ptr(mont)) {
+       goto err;
+    }
+    if (!BN_MONT_CTX_set(regenerated, mod, ctx)) {
+      TEST_error("failure to compute correct answer");
+      goto err;
+    }
+
+    /* Check equality of all components */
+    if (!TEST_int_eq(regenerated->ri, mont->ri)) {
+       goto err;
+    }
+    if (!TEST_BN_eq(&regenerated->RR, &mont->RR)) {
+      goto err;
+    }
+    /*  TODO: properly handle bignum case */
+    if (regenerated->n0[0] != mont->n0[0]) {
+      TEST_error("mismatch in n0[0]: expected: " BN_DEC_FMT1
+                 " got: " BN_DEC_FMT1, regenerated->n0[0], mont->n0[0]);
+      goto err;
+    }
+    if (regenerated->n0[1] != mont->n0[1]) {
+      TEST_error("mismatch in n0[1]");
+      goto err;
+    }
+    if (regenerated->flags != mont->flags) {
+      TEST_error("mismatch in flags");
+      goto err;
+    }
+    ret = 1;
+ err:
+    BN_MONT_CTX_free(regenerated);
+    return ret;
+}
+
+static int montgomery_correctness_test(EC_GROUP *group)
+{
+    int ret = 0;
+    BN_CTX *ctx = NULL;
+
+    ctx = BN_CTX_new();
+    if (ctx == NULL)
+      return ret;
```

Comment:
The indentation still looks wrong here throughout this test

Reply:
Oops, i only fixed the outermost layer. Time to go install the correct style across all buffers and try again